app.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const { getTopics } = require('./controllers/controllers')
+
+const app = express();
+app.use(express.json())
+//next(err)
```

Comment:
Please remove any comments in your code, you want to keep your code looking nice and tidy!